test/LendToAaveMigrator.t.sol

```diff
@@ -14,7 +14,7 @@ contract LendToAaveMigratorTest is Test {
     IERC20 public constant LEND = IERC20(0x80fB784B7eD66730e8b1DBd9820aFD29931aab03);
     uint256 public constant LEND_AAVE_RATIO = 100;
 
-    uint256 public constant lendAmountToMigrate = 8007719287288096435418;
+    uint256 public constant lendAmountToMigrate = 8007719287288096435418 + 841600717506653731350931;
```

Comment:
Why the new addition?

Reply:
we were not taking into account also the amount of lend to lend contract amount to reascue, which is used on the actual payload, so added it also on the test.